src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+server.port=8081
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/suriken
+spring.datasource.username=postgres
+spring.datasource.password=cetvorka4
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
pitanje za intervju: Sta sve moze da se stavi ovde, i cemu sluze ostale opcije pored create-drop

Reply:
E dobro pitanje. Verovatno klasik? Citao sam o tome. Vrednosti ovog properija mogu da budu: crate, create-drop, update, none i validate. 
Update ne brise tabele niti redove u njima, nego kada se pokrene server gleda koje su razlike trenutne seme u odnosu na prethodnu i dodaje ih. 
Create i create-drop su slicni, znaci pri pokretanju prvo obrisu postojecu semu, pa naprave novu, znaci brisu sve podatke. Najcesce se koriste u testiranju. Create-drop brise bazu kada se izvrse sve operacije.
None se koristi u produkciji. 
Validate samo proverava da li tabele i kolone postoje, u suprotnom baca exception.

Reply:
To